modules/programs/convert_img/test_basic.py

```
@@ -0,0 +1,225 @@
+    
+    print(f"\n--- Test Results ---")
+    print(f"Passed: {passed}")
+    print(f"Failed: {failed}")
+    print(f"Total:  {passed + failed}")
+    
+    if failed == 0:
+        print("\n🎉 All tests passed!")
+        return 0
+    else:
+        print(f"\n❌ {failed} test(s) failed!")
+        return 1
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)